### DISCUSSION TITLE
SFH & Dust extinction

### DISCUSSION THREAD

**Discussion by AmedeoGaudio88** [mboquien/cigale#99] [Q&A]
Could you explain to me how the SFR of a galaxy sample given as input to the software is calculated?
I ask you this because we tried to plot the SFR as a function of stellar mass and the plot was suspect.
Furthermore, is it possible to calculate dust extinction with CIGALE? How ?
Thanks for your availability.
 
Best regards

**Comment by mboquien** [comment]
See the email I sent you an hour ago.
